ignite/pkg/cosmosutil/genesis.go

```diff
@@ -6,17 +6,29 @@ import (
 	"crypto/sha256"
 	"encoding/hex"
 	"encoding/json"
+	"fmt"
 	"io"
 	"net/http"
 	"os"
+	"strconv"
+	"strings"
 	"time"
 
+	"github.com/buger/jsonparser"
 	"github.com/pkg/errors"
 )
 
 const (
-	genesisTimeField = "genesis_time"
-	chainIDField     = "chain_id"
+	SPNChainID                       = "spn-1"
```

Comment:
We should have a flag for this with default to `spn-1` since eventually you would connect with a network with a different chain ID